Source/Core/Core/HW/WiimoteEmu/EmuSubroutines.cpp

```diff
@@ -215,22 +211,6 @@ void Wiimote::RequestStatus(const wm_request_status* const rs)
   // status values
   *reinterpret_cast<wm_status_report*>(data + 2) = m_status;
 
-  // hybrid Wiimote stuff
-  if (WIIMOTE_SRC_REAL & g_wiimote_sources[m_index] && (m_extension->switch_extension <= 0))
-  {
-    using namespace WiimoteReal;
```

Comment:
Are those `WiimoteReal.h` includes still needed with all those removals? Goes for all the `WiimoteEmu` files changed in this commit.